### DISCUSSION TITLE
Where to find the scripts

### DISCUSSION THREAD

**Discussion by ThoomasNL** [yasenstar/ArchiSurance_Practice#3] [Q&A] [answered]
Hi, Thanks for the very interesting course. Where can I find the scripts you were showing, for example the one to distribute relations?

**Comment by yasenstar** [comment, marked as answer]
Hi @ThoomasNL,

A good resource while learning and using Archi tool is to explore [Archi Forum](https://forum.archimatetool.com/) often.

I actually found this nice Relationship Distribution script in the Forum (under jArchi section), which was contributed by William Malabry (credit to William!), here is the direct link to that post: [https://forum.archimatetool.com/index.php?topic=1174.msg6254#msg6254](https://forum.archimatetool.com/index.php?topic=1174.msg6254#msg6254).

My practice is to keep collecting those useful scripts and save to my local as one library for using, which you saw during the demo.

Furthermore, it is more than welcome that you may contribute back to the forum with your knowledge and insights.

Good luck, regards,
Xiaoqi

**Reply by ThoomasNL** [reply]
Thanks for the quick repy, Xiaoqi.
